modules_core/org.openbravo.client.application/src/org/openbravo/client/application/process/BaseProcessActionHandler.java

```diff
@@ -167,6 +169,12 @@ protected final JSONObject execute(Map<String, Object> parameters, String conten
     }
   }
 
+  private JSONObject doRefreshParent(JSONObject result) throws JSONException {
+    boolean deRefresh = !result.has(REFRESH_PARENT) || result.getBoolean(REFRESH_PARENT);
```

Comment:
result.has(REFRESH_PARENT) && result.getBoolean(REFRESH_PARENT)   ?

Reply:
The main idea is to have the refresh on true as default if it's not set. This is:
 - If the check existed, I need to set it as it came. So, the first check is to return true if it doesn't exist. As it's an 'or' and Java has lazy validations, the first true will return.
 - The second part of the 'or', is, in case the refresh exists, so if the refresh existed I need to use the boolean that was set, so I "false" the first condition to valid the boolean with the second one